Jenkinsfile

```diff
@@ -1,49 +1,20 @@
 #!groovy
-@Library('waluigi@v5.0.0') _
-
-standardProperties()
-
-node("primary") {
-  echo "Clean workspace"
-  cleanWs()
-
-  stage("checkout") {
-    checkout localBranch(scm)
-  }
-
-  stage("dependencies") {
-    yarnInstall()
-  }
-
-  stage("stamp") {
-    sh "yarn beehive-flow stamp"
-  }
-
-  stage("build") {
-    sh "yarn build"
-  }
-
-  stage("lint") {
-    sh "yarn lint"
-  }
-
-  bedrockBrowsers(testDirs: [ "src/test/ts/atomic", "src/test/ts/browser" ])
-
-  stage("storybook") {
-    def status = beehiveFlowStatus();
-    if (status.branchState == 'releaseReady' && status.isLatest) {
-      sshagent (credentials: ['3e856116-029e-4c8d-b57d-593b2fba3cb2']) {
-        sh 'yarn storybook-to-ghpages'
+@Library('waluigi@v6.0.0-rc') _
+
+beehiveFlowBuild(
+  test: {
+    bedrockBrowsers(testDirs: [ "src/test/ts/atomic", "src/test/ts/browser" ])
+  },
+  customSteps: {
+    stage("update storybook") {
```

Comment:
@jscasca should this run when the tests fail? If not then a guard will be needed here to check if the build is unstable.

Reply:
See https://github.com/tinymce/waluigi/pull/19/files#diff-9419e72a9aa81bc641a19ef34713e50110b925a24f0630538d0e797e46ce2711

Reply:
This is no longer necessary as we've changed it to never run these steps if the build is unstable.